be/src/exec/workgroup/work_group.cpp

```diff
@@ -63,9 +62,21 @@ namespace {
 class DefaultWorkGroupInitialization {
 public:
     DefaultWorkGroupInitialization() {
-        auto default_wg = std::make_shared<WorkGroup>("default_wg", 1, 10, 10, 10, WorkGroupType::WG_DEFAULT);
+        auto default_wg = std::make_shared<WorkGroup>("default_wg", 0, 1, 1, 1, WorkGroupType::WG_DEFAULT);
```

Comment:
Why is 0,1,1,1 ?

Reply:
This is just for test. Let's rebase the master branch with #2688, and remove it.